src/stan-users-guide/odes.Rmd

```diff
@@ -1,5 +1,54 @@
 # Ordinary Differential Equations  {#ode-solver.chapter}
 
+Stan provides three different built in methods for solving systems of
+ordinary differential equations (ODEs). If care is not taken, ODE models
```

Comment:
I'd start with a positive comment, à la "Stan has been widely used to fit ODE-based models and the interface is fairly straightforward to use. Still, it is worth noting that such models can be challenging to fit and that some care should be taken to build a smooth modeling workflow."

My primary concern is not discouraging first-time users who want to fit their first ODE-based model.